### ISSUE TITLE
fecha de nacimiento

### ISSUE THREAD

**Issue by jhannavarro** [arca-inc/koulu#5]
se puede crear un docente o estudiante poniendo una fecha de nacimiento poniendo el año ya sea superior al que estamos o si se es mayor de edad, obviamente todos los docentes tienen cedula y se puede  poner una fecha de  nacimiento inferior a los 18 años 
![1](https://user-images.githubusercontent.com/16363979/34795046-bc2693b8-f61e-11e7-9c52-
![1](https://user-images.githubusercontent.com/16363979/34795067-ccdcfb5c-f61e-11e7-9fa3-6d01d159f620.PNG)

**Comment by Nox04** [comment]
Sí, es un problema de validación. Hay que configurarlo en el template.
